packages/vaadin-time-picker/src/vaadin-time-picker.d.ts

```diff
@@ -177,3 +177,13 @@ declare global {
 }
 
 export { TimePicker };
+
+/**
+ * @deprecated Import `TimePicker` from `@vaadin/vaadin-time-picker` instead.
+ */
+export type TimePickerElement = TimePicker;
```

Comment:
TimePicker is a different scenario, where the package has not been moved yet, but still has renamed the element.

I decided to add the legacy alias to the actual components code for now. We should separate this, when we move the package.